developer/src/kmc-generate/test/fixtures/lexical-model/sample.en.sample/README.md

```diff
@@ -0,0 +1,27 @@
+Sample Project lexical model
+===================
+
+Description
+-----------
+# A mighty description
+
+Links
+-----
+
+Copyright
+---------
+See [LICENSE.md](LICENSE.md)
+
+Supported Platforms
+-------------------
+ * Windows
+ * macOS
+ * Linux
+ * Web
+ * iPhone
+ * iPad
+ * Android phone
+ * Android tablet
+ * Mobile devices
+ * Desktop devices
+ * Tablet devices
```

Comment:
Maybe this is carry-over from Developer template projects.
I thought lexical-models are only supported on mobile platforms.

Reply:
Good catch. I'll address this as a follow-up.